README.md

```diff
@@ -243,3 +243,66 @@ A more generic Machine Learning DApp that illustrates how to use the [m2cgen (Mo
 Demonstrates how to handle ERC-20 deposits and withdrawals.
 The application parses ERC-20 deposits received from the Portal and emits a notice confirming receipt.
 It then issues corresponding vouchers to return the assets back to the depositor.
+
+## Common pitfalls while using rollups. 
+
+Here are some tips for the most commom problems while building, testing, and deploying the rollups dapps.
+
+### Getting wrong values in the graphql notices
+
+Sometimes we will be testing more than one DApp in a host machine. In this scenario we can sometimes forget to run `docker-compose -f ../docker-compose.yml -f ./docker-compose.override.yml down -v` between the tests, or when it is mentioned in the previous documentation. It is recommended to always run this while changing the DApp we are using. Some known problems this can cause are:
+- Wrong array of notices
+- Blank array of notices
+- inconsistent epoch of blockchain
+
+### Small diferences in commands
+
+Some commands are similar but not equal. For instance, while building an DApp for testing, we have `docker buildx bake --load` which builds the DApp for us. While Deploying an DApp, the command is similar but it adds the tag `machine` in : `docker buildx machine bake --load`. There are some other commands, so be sure to check the right command for the right situation.
+
+### Run the backend for the hostmode
+
+While testing, we can easily forget to run the enviroment step, which causes the dapp not working properlly. In host mode running the enviroment part is the backend itself, so if it isn't executed it will not work.
```

Comment:
Typo: "**environment**" instead of "enviroment", "**properly**" instead of "properlly".